CHANGES.rst

```diff
@@ -23,6 +23,11 @@ Unreleased
 * :meth:`.Comment.parent` to obtain the parent of a :class:`.Comment`.
 * :meth:`.opt_in` and :meth:`.opt_out` to :class:`.Subreddit` to permit working
   with quarantined subreddits.
+* :class:`praw.models.LiveUpdate`.
+* :meth:`praw.models.LiveUpdate.thread` to get the parent live thread of the
+  update.
+* :meth:`praw.models.LiveThread.__getitem__` to get the update of the thread.
```

Comment:
I'd prefer to avoid linking to `__` methods in the changelog. Could you instead give an example like:

> Ability to access an individual LiveUpdate via ``reddit.live('THREAD_ID')['UPDATE_ID']``.